share/cpt/acton.cpt

```diff
@@ -3,10 +3,10 @@
 #
 #	www.fabiocrameri.ch/visualisation
 #
-# License: Creative Commons Attribution 4.0 International License
-# Copyright (c) 2020, Fabio Crameri All rights reserved.
-# Crameri, F., (2020). Scientific colour maps. Zenodo. https://zenodo.org/record/4153113
-# This is Scientific Colour Maps version 6.0.4
+# License: MIT License
+# Copyright (c) 2021, Fabio Crameri. All rights reserved.
```

Comment:
```suggestion
# Copyright (c) 2021, Fabio Crameri.
```
In looking at the license in the zenodo archive, the 'all rights reserved' part is not included in the license. Perhaps this part was left over from the CC 4.0 license?

Reply:
You're right, I'll take out the All rights reserved.

![image](https://user-images.githubusercontent.com/23487320/108779823-5d0f6400-75cc-11eb-93ee-da33741cf93f.png)